pkg/config/config.go

```diff
@@ -11,8 +11,9 @@ import (
 	"github.com/spf13/pflag"
 )
 
-// DefaultConfigPath specifies the default location of the STI config file
-const DefaultConfigPath = ".stifile"
+// DefaultConfigPath specifies the default location of the S2I config file
+const DeprecatedConfigPath = ".stifile"
```

Comment:
i'm not a fan of having these 'Deprecated*' constants... we are not doing that of other deprecated stuff and we will expose them to users as API (where there is no point exposing them).

I would have just one constant and handle deprecation below, what do you think?